Dockerfile

```diff
@@ -1,22 +1,53 @@
-#
-# Dockerfile for Storm Mesos framework
-#
-FROM mesosphere/mesos:0.22.0-1.0.ubuntu1404
-MAINTAINER Timothy Chen <tnachen@gmail.com>
+#this is oracle java8 atop phusion baseimage
+FROM opentable/baseimage-java8:latest
 
-# build packages
-ENV DEBIAN_FRONTEND noninteractive
-RUN apt-get update
-RUN apt-get install -yq openjdk-7-jdk
 
-# export environment
-ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64
 
-ENV MESOS_NATIVE_JAVA_LIBRARY /usr/lib/libmesos.so
 
-# copy local checkout into /opt/storm
-ADD . /opt/storm
+#prep to install libmesos.so
+# (this step varies from mesos instructions based on http://unix.stackexchange.com/questions/75892/keyserver-timed-out-when-trying-to-add-a-gpg-public-key)
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv E56151BF
 
-WORKDIR /opt/storm
+#below echo | tee command is taken from mesos instructions
+#DISTRO=$(lsb_release -is | tr '[:upper:]' '[:lower:]')
+#CODENAME=$(lsb_release -cs)
+#echo "deb http://repos.mesosphere.io/${DISTRO} ${CODENAME} main" | \
+#  sudo tee /etc/apt/sources.list.d/mesosphere.list
+RUN echo "deb http://repos.mesosphere.io/$(lsb_release -is | tr '[:upper:]' '[:lower:]') $(lsb_release -cs) main" | tee /etc/apt/sources.list.d/mesosphere.list
+RUN cat /etc/apt/sources.list.d/mesosphere.list
+
+#to make mesos repo available and install mesos:
+RUN apt-get update && apt-get install -y \
+    mesos \
+    python
+
+#install storm
+#inspired by https://github.com/wurstmeister/storm-docker/blob/master/storm/Dockerfile
+RUN wget -q -O - http://mirrors.sonic.net/apache/storm/apache-storm-0.9.4/apache-storm-0.9.4.tar.gz | tar -xzf - -C /opt
```

Comment:
If you remove the build release stuff and move it here, then users will have to use docker then right?
I think we should make this optional